src/models/searchservice/response/HighlightedFields.ts

```diff
@@ -0,0 +1,47 @@
+import { HighlightedValue } from './HighlightedValue';
+
+/**
+ * A mapping of the values emphasized by the Answers API.
+ *
+ * @remarks
+ * The structure of HighlightedFields matches the structure of the corresponding rawData of a {@link Result}.
```

Comment:
I think it's important to caveat this. The structure of `HighlightedFields` doesn't necessarily match the structure of `rawData`. Not every field that appears in `rawData` will have highlighting. The fields that do have highlighting will have the same structure in here that they do in `rawData`.

Reply:
agree!